scripts/install_third_party.py

```diff
@@ -27,6 +27,11 @@
 
 import common
 
+#set the proxy to empty sting if the installation is through Vagrant so that
```

Comment:
Thanks @seanlip 
I will just paste what you have given in the note if you don't mind :smile: